ceph/rbd/pkg/provision/rbd_util.go

```diff
@@ -28,7 +30,8 @@ import (
 )
 
 const (
-	imageWatcherStr = "watcher="
+	imageWatcherStr       = "watcher="
+	defaultCommandTimeOut = 5
```

Comment:
How about to make it configurable? If someone want a shorter or longer timeout.

Reply:
Sounds good. Let me add it.